Agregadas definiciones

Olarruz opened the pull request:
Se agregaron definiciones, se corrigieron los todo, y se comenzó a seguir la estructura del .bat

Olarruz committed:
Agregadas definiciones

Se agregaron definiciones, se corrigieron los todo, y se comenzó a seguir la estructura del .bat

luis-telesforo commented on texto-1.pdf:
Esto no debería estar aquí

luis-telesforo commented on texto-1.tex:
```diff
@@ -22,25 +22,60 @@
+		$0_A$ es un elemento de $A$
+		\item [3.]
+		* es una función cuyo dominio es $A \times A$, y cuyo contradominio es $A$, y que cumple que:
+		\begin{enumerate}
```
Igual aquí, los enumerate anidados son inteligentes

luis-telesforo commented on texto-1.tex:
```diff
@@ -22,25 +22,60 @@
 \newtheorem{proposition}[theorem]{Proposición}
 \newtheorem{example}[theorem]{Ejemplo}
 \newtheorem{definition}[theorem]{Definición}
+\newtheorem{reminder}[theorem]{Recordatorio}
```
No es necesario esto, un párrafo en prosa es suficiente.

luis-telesforo commented on texto-1.tex:
```diff
@@ -22,25 +22,60 @@
+			\item [a)]
+			* es asociativa.
+			\item [b)]
+			Para toda $a\in A$ se cumple que $*(0_A,a)=*(a,0_A)=a$.
```
No quieren usar esta notación, no es usual nunca.

luis-telesforo commented on texto-1.tex:
```diff
@@ -22,25 +22,60 @@
+	\end{reminder}
+	
+	\begin{reminder}
+		Sea $(A,*,0_A)$ un grupo. Decimos que $B$ es subrgupo de $A$ si ocurre que $B \subseteq A$, $B \neq \varnothing$, y 
```
Usen \emptyset en lugar de \varnothing

luis-telesforo commented on texto-1.tex:
```diff
@@ -22,25 +22,60 @@
+	\end{reminder}
+
+	\begin{notation}
+		A partir de ahora, a los grupos los denotaremos solo con su primer elemento, al elemento $0_A$ como $0$, a la función $*$ con la suma, y 
```
Esto no se entiende, en los grupos no hay primer elemento.

luis-telesforo reviewed (changes requested):
Me parece que no están respetando los 100 caracteres por línea.

Eliminen lo de reminder, sólo dificulta la lectura, tanto en el código como en el pdf.

Por favor, resuelvan mis comentarios.

Olarruz committed:
Resolución de comentarios

Resueltos algunos comentarios

luis-telesforo commented on texto-1.tex:
```diff
@@ -22,25 +22,62 @@
+	
+	\begin{definition}
+		Sea $(A,*,0_A)$ un grupo. Decimos que $B$ es subrgupo de $A$ si ocurre que $B \subseteq A$ y 
+		$(B,*_{\restriction A},0_A)$ es un grupo; y lo denotamos como $B \leq A$.
```
No era lo que esperaba, pero funciona. Hay un error de dedo, no debe ser la restricción a $$A$$ sino a $$B\times B$$

luis-telesforo commented on texto-1.tex:
```diff
@@ -22,25 +22,62 @@
+	\end{definition}
+
+	\begin{notation}
+		A partir de ahora, a un grupo $(A,+,0_A)$ lo denotaremos solo como $A$,
```
No vi esto la primera vez. No queda claro porqué cambiaron a la notación de suma. Además hay un error en la terna.

luis-telesforo commented on texto-1.tex:
```diff
@@ -22,25 +22,62 @@
+
+	\begin{notation}
+		A partir de ahora, a un grupo $(A,+,0_A)$ lo denotaremos solo como $A$,
+		al elemento $0_A$ como $0$, a la función $*$ con la suma, y denotaremos $\sum_{i=1}^{n} a = a^n$.
```
Esto es incorrecto, la suma no es la potencia.

luis-telesforo reviewed (changes requested):
Quedaron algunas cosas confusas de nuevo. Respecto al número de caracteres, es en el código, el pdf no lo están controlando ustedes.

Olarruz committed:
Correciones pequeñas

Se corrigieron algunos errores de dedo, y se cambió el formato para que no hubiese demasiados caracteres por línea

luis-telesforo reviewed (approved):
Muy bien. Ya pueden mezclar esto.
